packages/x-components/src/x-modules/search/store/actions/fetch-and-save-search-response.action.ts

```diff
@@ -29,6 +39,7 @@ const { fetchAndSave, cancelPrevious } = createFetchAndSaveActions<
       commit('setFacets', facets);
     }
 
+    commit('setQueryTagging', queryTagging);
```

Comment:
I would like to see this new info in the `fetchAndSaveSearchResponse` action test, please update it.